tests/v2/validation/provisioning/ssh.go

```diff
@@ -0,0 +1,26 @@
+package provisioning
+
+// create all tests here that use executable tests
+
+import (
+	"strconv"
+
+	"github.com/rancher/rancher/tests/framework/pkg/nodes"
+)
+
+// Test to check cpu usage
+
+func CheckCPU(node *nodes.Node) (string, error) {
+	command := "ps -C agent -o %cpu --no-header"
+	output, err := node.ExecuteCommand(command)
+	if err != nil {
+		return output, err
+	}
+
+	output_int, err := strconv.Atoi(output)
+	if output_int > 30 {
```

Comment:
I would say you could just define a `const` block in this `ssh.go` file and just reference that way. Since additional tests are going to be made, it's possible more hard-code values will come, so we can just have that `const` block now.